README.md

```diff
@@ -31,6 +31,12 @@ rust-openssl to a separate installation. OSX releases starting at 10.11, "El
 Capitan", no longer include OpenSSL headers which will prevent the `openssl`
 crate from compiling.
 
+Try the following commands to fix the issue (assumes Homebrew is used for package management):
+```
+brew install openssl
+brew link openssl --force
```

Comment:
I am not sure I'm on board with doing this - there is a reason that brew does not link openssl by default. A better recommendation could be to set `OPENSSL_LIB_DIR` to `/usr/local/opt/openssl/lib` and `OPENSSL_INCLUDE_DIR` to `/usr/local/opt/openssl/include`.